frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -354,6 +354,9 @@ def get_build_task(task_id):
     try:
         task = BuildsLogic.get_build_task(task_id)
         build_record = get_build_record(task)
+        if build_record is None:
```

Comment:
I can see that `get_build_record` and `get_srpm_build_record` are only used in API endpoint. Can we perhaps change return type of these to `dict` from `Optional[dict]`? Backend thinks these endpoints won't return `null` but they do

Reply:
I can't reproduce the `null` output, I can't decide.  It seems completely OK to return 404 if no such build task exists, 500 upon error, 400 for invalid requests.  And return 200 if and only if the request was processed.

Reply:
the build that causes this error is https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/build/5942294/

namely, the f36-s390x chroot, as you can see there the chroot for this build does not exist... it looks like it was never created in DB for this build - I can see that this build was created at the same time I made f36 chroots EOL so some rare unlucky race condition perhaps :D

Until that time the build is still in a pending state and it is flooding our logs with errors (cca 2 months).

Reply:
Hmmm, that would make sense, good catch.